shotover-proxy/src/frame/cassandra.rs

```diff
@@ -205,38 +205,45 @@ impl CassandraFrame {
                 if let ResponseBody::Result(result) = frame.response_body()? {
                     match result {
                         ResResultBody::Rows(rows) => {
-                            let converted_rows =
-                                if rows.metadata.flags.contains(RowsMetadataFlags::NO_METADATA) {
-                                    rows.rows_content
-                                        .into_iter()
-                                        .map(|row| {
-                                            row.into_iter()
-                                                .map(|row_content| {
+                            let converted_rows = if rows
+                                .metadata
+                                .flags
+                                .contains(RowsMetadataFlags::NO_METADATA)
+                            {
+                                rows.rows_content
+                                    .into_iter()
+                                    .map(|row| {
+                                        row.into_iter()
+                                            .map(|row_content| {
+                                                if row_content.is_empty() {
```

Comment:
Can you explain this change?
Shouldnt Null be handled at a lower level?

Reply:
The Datastax driver doesn't seem to let you create unbound prepared statements and this causes that error (tries to unwrap on a null value). I'm seeing if I can fix it by finishing off the scylla test helpers.